invoiceregistry_test.go

```diff
@@ -128,70 +125,42 @@ func TestInvoiceExpiry(t *testing.T) {
 
 	c := newRegistryTestContext(t, false)
 
-	// Subscribe to updates for invoice 1.
-	updateChan1, cancel1 := c.subscribe(1)
-
 	// Add invoice.
-	c.addInvoice(1, time.Second)
-
-	// Expect an open notification.
-	update := <-updateChan1
-	require.Equal(t, persistence.InvoiceStateOpen, update.State)
-
-	// Expected an expired notification.
-	update = <-updateChan1
-	require.Equal(t, persistence.InvoiceStateCancelled, update.State)
-	require.Equal(t, persistence.CancelledReasonExpired, update.CancelledReason)
-
-	cancel1()
-
-	// Add another invoice.
-	c.addInvoice(2, time.Second)
-
-	// Expect the open update.
-	updateChan2, cancel2 := c.subscribe(2)
-	update = <-updateChan2
-	require.Equal(t, persistence.InvoiceStateOpen, update.State)
-	cancel2()
-
-	// Stop the registry.
-	c.stop()
+	invoice, preimage := c.createInvoice(1, time.Second)
 
 	// Wait for the invoice to expire.
 	time.Sleep(2 * time.Second)
```

Comment:
pre: you could use smaller durations in order to make tests run faster?

Reply:
Ah yes, that's true in this case. Fixed.